nikola/post.py

```diff
@@ -745,11 +745,16 @@ def section_color(self, lang=None):
 
     def section_link(self, lang=None):
```

Comment:
That’s a good question. I just fixed the existing code (by @Aeyoun). I looked a bit further, and it seems that the Post  class does not have access to the Nikola site object, and therefore cannot talk to `site.link()` nor to the plugin manager.